qt/scientific_interfaces/Inelastic/Processor/MomentsPresenter.cpp

```diff
@@ -129,7 +124,7 @@ void MomentsPresenter::setFileExtensionsByName(bool filter) {
 /**
  * Handle when Run is clicked
  */
-void MomentsPresenter::handleRunClicked() { runTab(); }
+void MomentsPresenter::handleRun() { runTab(); }
```

Comment:
It would be great to move the code from the `run()` method into here so we no longer need to call `runTab()` from the parent class, and the same in the other Presenter classes for this interface. We can then remove the `run()` member functions